handlers/handlers.go

```
@@ -352,6 +353,16 @@ func editionsList(w http.ResponseWriter, req *http.Request, dc DatasetClient, re
 		}
 	}
 
+	numberOfEditions := len(datasetEditions)
+	if numberOfEditions == 1 {
+		var latestVersionURL, err = url.Parse(datasetEditions[0].Links.LatestVersion.URL)
+		if err != nil {
```

does this need to return an error status at this point?